physicalplan/aggregate/vector_table.go

```diff
@@ -88,32 +86,6 @@ func newVectorAccumulator(expr parser.ItemType) (vectorAccumulator, error) {
 		return func(in []float64) float64 {
 			return floats.Sum(in) / float64(len(in))
 		}, nil
-	case "stddev":
-		return func(in []float64) float64 {
-			var count float64
-			var mean, cMean float64
-			var aux, cAux float64
-			for _, v := range in {
-				count++
-				delta := v - (mean + cMean)
-				mean, cMean = scan.KahanSumInc(delta/count, mean, cMean)
-				aux, cAux = scan.KahanSumInc(delta*(v-(mean+cMean)), aux, cAux)
-			}
-			return math.Sqrt((aux + cAux) / count)
-		}, nil
-	case "stdvar":
-		return func(in []float64) float64 {
-			var count float64
-			var mean, cMean float64
-			var aux, cAux float64
-			for _, v := range in {
-				count++
-				delta := v - (mean + cMean)
-				mean, cMean = scan.KahanSumInc(delta/count, mean, cMean)
-				aux, cAux = scan.KahanSumInc(delta*(v-(mean+cMean)), aux, cAux)
-			}
-			return (aux + cAux) / count
-		}, nil
```

Comment:
We don't need the 2 functions anymore?

Reply:
Since the implementation is the same as the scalar version, they should be covered here: https://github.com/thanos-community/promql-engine/pull/52/files#diff-0a6fbd7e1f59129205d1acd4db27f232b5d21f7e6327aa814ac4d062ae98bc6aR148 

Tests are also passing so I think this should work.